rasa/core/policies/ensemble.py

```diff
@@ -117,11 +118,14 @@ def train(
         self,
         training_trackers: List[DialogueStateTracker],
         domain: Domain,
+        interpreter: NaturalLanguageInterpreter,
         **kwargs: Any,
     ) -> None:
         if training_trackers:
             for policy in self.policies:
-                policy.train(training_trackers, domain, **kwargs)
+                policy.train(
+                    training_trackers, domain, interpreter=interpreter, **kwargs
+                )
```

Comment:
```suggestion
                number_of_arguments_in_rasa_1_0 = 3
                arguments = common_utils.arguments_of(policy.predict_action_probabilities)
                if (
                    len(arguments) > number_of_arguments_in_rasa_1_0
                    and "interpreter" in arguments
                ):
                    policy.train(
                        training_trackers, domain, interpreter=interpreter, **kwargs
                    )
                else:
                    common_utils.raise_warning(
                        "The function `train` of "
                        "the `Policy` interface was changed to support "
                        "additional parameters. Please make sure to "
                        "adapt your custom `Policy` implementation.",
                        category=DeprecationWarning,
                    )
                    policy.train(
                        training_trackers, domain, **kwargs
                    )
```

Reply:
we don't have to change `train` as the `Policy` interface already includes the `**kwargs` argument. Custom polices will  hence work out of the box.

Reply:
Ah, of course!